modules/renter/renter.go

```diff
@@ -232,6 +247,62 @@ func newRenter(cs modules.ConsensusSet, tpool modules.TransactionPool, hdb hostD
 	return r, nil
 }
 
+// managedMemoryAvailableAdd adds the amount provided to the renter's total
+// memory available.
+func (r *Renter) managedMemoryAvailableAdd(amt uint64) {
+	id := r.mu.Lock()
+	r.memoryAvailable += amt
+	r.mu.Unlock(id)
+}
```

Comment:
why make these `managed`?

Reply:
I felt that it made the code more readable. There are a few places where we lock, modify memAvail, and then unlock. I thought it was cleaner to just have one line of code there instead of 3. Also, we can add sanity checks and stuff later if we want that check for going below zero or above baseline, and it won't clutter the code.